db/migrate/20240317155844_add_filter_fields_to_project.rb

```diff
@@ -0,0 +1,10 @@
+class AddFilterFieldsToProject < ActiveRecord::Migration[7.0]
+  def change
+    add_column :projects, :in_home_pets, :string
+    add_column :projects, :has_smoke_in_home, :boolean, default: false
+    add_column :projects, :no_smoke, :boolean
+    add_column :projects, :no_cats, :boolean
+    add_column :projects, :no_dogs, :boolean
```

Comment:
Are these covered in the other fields?

Reply:
@swards These mean if set to true that the person who is providing the project does want someone to work on the project hat has smoke, cats, or dogs in the house.  The other `in_home_pets and` and `has_smoke_in_home` mean the project was in a home with pets listed in that string and there was or was not smoke in the house.  These match the exact the same properties on the finisher.  To me there is crazy complicated.  Could be as simple as storing has_pets, and smoke in home

Reply:
Yeah - I see.  I think they need both.  I think it's good they match with the types.  We can talk this one out when they start to filter or we auto-suggest finishers for projects.  A migration to change this would be easy to do.